frontend/src/router/pages/EditAndReview.jsx

```diff
@@ -29,67 +97,42 @@ function EditAndReview() {
             Status
           </label>
 
-          <select id="status" className="form-select">
-            <option value=""></option>
-          </select>
-        </div>
+          <select
+            id="status"
+            className="form-select"
+            value={statusFilter}
+            onChange={(e) => setStatusFilter(e.target.value)}
+          >
+            <option value="">All</option>
 
-        <div className="col-lg-3 col-md-6 col-12">
-          <label htmlFor="bundle" className="form-label">
-            Bundle
-          </label>
-
-          <select id="bundle" className="form-select">
-            <option value=""></option>
```

Comment:
After the keycloak implementation is done (has it?), we could add a function to check a user's role (e.g. Park Operator/Admin) and display "Bundle" and "Has reservation" in the table if a user has the admin role. We did some thing similar in the SP. 

https://github.com/bcgov/bcparks-staff-portal/blob/main/frontend-legacy/src/auth/PrivateElement.js
https://github.com/bcgov/bcparks-staff-portal/blob/main/frontend-legacy/src/components/page/appDashboard/AppDashboard.js

Reply:
The users table / auth stuff will be separate; I think we need to work out a way to pull a bunch of data from Strapi first. Right now you have to log in with your IDIR, but nothing is checking roles or any other permissions.